docker-compose-deployment/docker-compose.yaml

```diff
@@ -24,16 +24,16 @@ services:
       - 'payments:payments'
     ports:
       - '8080:8080'
-  product-api:
-    image: 'hashicorpdemoapp/product-api:v0.0.20'
-    volumes:
-      - type: bind
-        source: ./conf.json
-        target: /conf.json
-    links:
-      - 'product-db:product-db'
-    ports:
-      - '9090:9090'
+  # product-api:
+  #   image: 'hashicorpdemoapp/product-api:v0.0.20'
+  #   volumes:
+  #     - type: bind
+  #       source: ./conf.json
+  #       target: /conf.json
+  #   links:
+  #     - 'product-db:product-db'
+  #   ports:
+  #     - '9090:9090'
```

Comment:
@im2nguyen if we don't need this, can we remove the lines please.

Reply:
i accidentally merged the wrong file! my bad